tools/driver/p4c.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+from p4c.main import main
+
+if __name__ == '__main__':
+    main()
```

Comment:
after installation `python setup.py install`, user is able to call `p4c` to invoke the driver.